packages/admin-panel/src/dataTables/useParams.js

```diff
@@ -32,25 +34,30 @@ export const useParams = ({ recordData, onEditField }) => {
       [...newAdditionalParams, ...builtInParams].map(p => [p.name, undefined]),
     );
     setRuntimeParams(defaultRuntimeParams);
-  }, [JSON.stringify(config?.additionalParams), recordData.type]);
+  }, [recordData.type]);
+
+  const onEditAdditionalParams = newAdditionalParams => {
+    onEditField('config', {
+      ...config,
+      additionalParams: newAdditionalParams,
+    });
+
+    setAdditionalParams(newAdditionalParams);
+  };
```

Comment:
For `additionalParams` state, I tried to replace it with redux state (recordData), but the state management and performance is not well as useState(). 

Basically additionalParams just have one more config (id) in each additional parameter than (recordData).